superdesk/default_settings.py

```diff
@@ -588,6 +588,9 @@ def local_to_utc_hour(hour):
 DEFAULT_GENRE_VALUE_FOR_MANUAL_ARTICLES = env('DEFAULT_GENRE_VALUE_FOR_MANUAL_ARTICLES',
                                               [{'qcode': 'Article', 'name': 'Article (news)'}])
 
+#: Defines default value for category for ingested and auto published articles
+DEFAULT_CATEGORY_VALUE_FOR_AUTO_PUBLISHED_ARTICLES = env('DEFAULT_CATEGORY_VALUE_FOR_AUTO_PUBLISHED_ARTICLES', None)
```

Comment:
I wouldn't try to read it from env given that it should be list of objects, also it would make sense to have that hardcoded value as default here

Reply:
What about using a comma separated list of qcodes. When assigning the value retrieve the list of actual CV objects (otherwise it may not get the correct name if the CV is changed).